server/src/swagger.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
this file is not needed after migrating server to nest

Reply:
remove swagger.json